src/uwtools/rocoto.py

```diff
@@ -8,7 +8,7 @@
 from typing import Any, List, Optional, Tuple, Union
 
 from lxml import etree
-from lxml.etree import Element, SubElement
+from lxml.etree import Element, SubElement, _Element
```

Comment:
It turns out that `_Element` is the name of the _type_ defined my `lxml` for XML element objects, while `Element` is a function that creates elements:
```
>>> import lxml.etree
>>> type(lxml.etree._Element)
<class 'type'>
>>> type(lxml.etree.Element)
<class 'cython_function_or_method'>
```
So, `_Element` needs to be used in type hints.